exercises/practice/darts/.docs/instructions.md

```diff
@@ -6,12 +6,12 @@ Write a function that returns the earned points in a single toss of a Darts game
 
 In our particular instance of the game, the target rewards 4 different amounts of points, depending on where the dart lands:
 
-- If the dart lands outside the target, player earns no points (0 points).
-- If the dart lands in the outer circle of the target, player earns 1 point.
-- If the dart lands in the middle circle of the target, player earns 5 points.
-- If the dart lands in the inner circle of the target, player earns 10 points.
+- If the dart lands outside the target, the player earns no points (0 points).
+- If the dart lands in the outer circle of the target, the player earns 1 point.
+- If the dart lands in the middle circle of the target, the player earns 5 points.
+- If the dart lands in the inner circle of the target, the player earns 10 points.
 
-The outer circle has a radius of 10 units (this is equivalent to the total radius for the entire target), the middle circle a radius of 5 units, and the inner circle a radius of 1.
```

Comment:
I think these instructions are from the original example instructions and we shouldn't update them.

Reply:
Revert the changes in https://github.com/exercism/ballerina/pull/217/commits/c501edc9be80fb565132e69ca45b641070fb9fd3